website/content/en/docs/contribution-guidelines/releasing.md

```diff
@@ -198,12 +198,10 @@ We will use the `v1.3.1` release version in this example.
 
 #### 0. Lock down release branches on GitHub
 
-Lock down the `v1.3.x` branch to prevent further merges/commits.
-
-To do this, edit the `Branch protection rules`: https://github.com/operator-framework/operator-sdk/settings/branches
-
-- click `Edit` on the `v.*` branch rule.
-- In section `Protect matching branches` of the `Rule settings` box, set "Required approving reviewers" to `6`.
+1. Lock down the `v1.3.x` branch to prevent further commits before the release completes:
+  1. Go to `Settings -> Branches` in the SDK repo.
```

Comment:
Nit: I think the formatting here didn't quite do what you intended with the indents

Reply:
Yeah, running the docs site locally it doesn't look like it formatted correctly. That being said, I pretty much snatched this straight from an existing section for locking down the branch and just changed some values. The section I snatched from:
https://github.com/operator-framework/operator-sdk/blob/cbeec475e4612e19f1047ff7014342afe93f60d2/website/content/en/docs/contribution-guidelines/releasing.md?plain=1#L77-L80